WordPress/src/main/java/org/wordpress/android/ui/stats/models/SingleItemModel.java

```diff
@@ -25,7 +25,14 @@ public SingleItemModel(String blogId, String date, String itemID, String title,
         this.mItemID = itemID;
         this.mTitle = title;
         this.mTotals = totals;
-        this.mUrl = url;
+
+        // We could get invalid data back from the server. Check that URL is OK.
+        if (url == null || "null".equals(url) || "false".equals(url)) {
```

Comment:
Should we use https://developer.android.com/reference/android/webkit/URLUtil.html#isValidUrl(java.lang.String) instead?